src/script.c

```diff
@@ -141,6 +143,11 @@ void environment_exit(environment_t *env) {
 }
 
 bool execute_script(const char *name, environment_t *env) {
+	if(!enable_scripts) {
```

Comment:
We might still want to allow `tinc-up` to be executed; it's run before privileges are dropped. This might make it easier for people who are used to the `tinc-up` script to configure the network.

Reply:
Hmm. I'm not sure how to reconcile that with the new option. If a user disables script execution, she expects them not to run, no?

Maybe we could remove the option altogether, and add another one that configures the isolation level. Such as:

`Sandbox = off | medium | strong`

- `off`: tincd continues working as it does now; if `medium` is implemented properly, this shouldn't be needed by anyone.
- `medium`: used by default; on OpenBSD it would probably be a call to `pledge()` and nothing else.
- `strong`: most scripts don't work, with the exception of `tinc-up`; exec proxies won't run; probably some other limitation I can't think of right now.

Reply:
That sounds great!